script/ReviewsData.json

```diff
@@ -0,0 +1,32 @@
+[
+    {
+      "rating": 1,
+      "body": "Nulla ut erat id mauris vulputate elementum. Nullam varius. Nulla facilisi.\n\nCras non velit nec nisi vulputate nonummy. Maecenas tincidunt lacus at velit. Vivamus vel nulla eget eros elementum pellentesque.\n\nQuisque porta volutpat erat. Quisque erat eros, viverra eget, congue eget, semper rutrum, nulla. Nunc purus."
```

Comment:
I think we need to also seed what products reviews belong to, and what users reviews belong to.

Reply:
added